bin/trait_mapping/create_table_for_manual_curation.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import argparse
+
+from eva_cttv_pipeline.trait_mapping.ols import (
+    get_ontology_label_from_ols, is_current_and_in_efo, is_in_efo,
+)
+
+
+def find_previous_mapping(trait_name, previous_mappings):
+    if trait_name not in previous_mappings:
+        return ''
+    uri = previous_mappings[trait_name]
+    label = get_ontology_label_from_ols(uri)
+    uri_is_current_and_in_efo = is_current_and_in_efo(uri)
+    uri_in_efo = is_in_efo(uri)
+    if uri_in_efo:
+        trait_status = 'EFO_CURRENT' if uri_is_current_and_in_efo else 'EFO_OBSOLETE'
+    else:
+        trait_status = 'NOT_CONTAINED'
+    trait_string = '|'.join([uri, label, 'NOT_SPECIFIED', 'previously-used', trait_status])
+    return trait_string
+
+
+def find_exact_mapping(trait_name, mappings):
+    for mapping in mappings:
+        if mapping.lower().split('|')[1] == trait_name:
+            return mapping
+    return ''
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--traits-for-curation',
+        help='Table with traits for which the pipeline failed to make a confident prediction')
+    parser.add_argument(
+        '--previous-mappings',
+        help='Table with all mappings previously issued by EVA')
+    parser.add_argument(
+        '--final-table-for-curation',
```

Comment:
The rest of our tools use `-o` and `--output`.

Reply:
Fair enough, fixed.